lib/streamlit/elements/streamlit_plotly_theme.py

```diff
@@ -0,0 +1,193 @@
+    [0.2222222222222222, SEQUENTIAL_12],
+    [0.3333333333333333, SEQUENTIAL_13],
+    [0.4444444444444444, SEQUENTIAL_14],
+    [0.5555555555555556, SEQUENTIAL_15],
+    [0.6666666666666666, SEQUENTIAL_16],
+    [0.7777777777777778, SEQUENTIAL_17],
+    [0.8888888888888888, SEQUENTIAL_18],
+    [1.0, SEQUENTIAL_19],
+]
+
+pio.templates["streamlit"] = go.layout.Template(
+    data=go.layout.template.Data(
+        candlestick=[
+            go.layout.template.data.Candlestick(
+                decreasing=go.candlestick.Decreasing(
+                    line=go.candlestick.decreasing.Line(color=DECREASING_32)
+                ),
+                increasing=go.candlestick.Increasing(
+                    line=go.candlestick.increasing.Line(color=INCREASING_31)
+                ),
+            )
+        ],
+        contour=[go.layout.template.data.Contour(colorscale=streamlit_colorscale)],
+        contourcarpet=[
+            go.layout.template.data.Contourcarpet(colorscale=streamlit_colorscale)
+        ],
+        heatmap=[go.layout.template.data.Heatmap(colorscale=streamlit_colorscale)],
+        histogram2d=[
+            go.layout.template.data.Histogram2d(colorscale=streamlit_colorscale)
+        ],
+        icicle=[
+            go.layout.template.data.Icicle(textfont=go.icicle.Textfont(color="white"))
+        ],
+        sankey=[
+            go.layout.template.data.Sankey(
+                textfont=go.sankey.Textfont(color=GRAY_70_35, family=BODY_FONT)
+            )
+        ],
+        scatter=[
+            go.layout.template.data.Scatter(
+                marker=go.scatter.Marker(line=go.scatter.marker.Line(width=0))
+            )
+        ],
+        table=[
+            go.layout.template.data.Table(
+                cells=go.table.Cells(
+                    fill=go.table.cells.Fill(color=BG_COLOR_37),
+                    font=go.table.cells.Font(family=BODY_FONT, color=GRAY_90_36),
+                    line=go.table.cells.Line(color=FADED_TEXT_05_38),
+                ),
+                header=go.table.Header(
+                    font=go.table.header.Font(family=BODY_FONT, color=GRAY_70_35),
+                    line=go.table.header.Line(color=FADED_TEXT_05_38),
+                    fill=go.table.header.Fill(color=BG_MIX_39),
+                ),
+            )
+        ],
+        waterfall=[
+            go.layout.template.data.Waterfall(
+                increasing=go.waterfall.Increasing(
+                    marker=go.waterfall.increasing.Marker(color=INCREASING_31)
+                ),
+                decreasing=go.waterfall.Decreasing(
+                    marker=go.waterfall.decreasing.Marker(color=DECREASING_32)
+                ),
+                totals=go.waterfall.Totals(
+                    marker=go.waterfall.totals.Marker(color=TOTAL_33)
+                ),
+                connector=go.waterfall.Connector(
+                    line=go.waterfall.connector.Line(color=GRAY_70_35, width=2)
```

Comment:
nit: since `width=2` is not color related, this needs to be defined here? Just want to make sure we are only adding the minimal aspects in our placeholder theme.

Reply:
Again, I just thought we can just plainly override and it likely won't be a big deal since we do that with the background colors and other such things in the layoutWithThemeDefaults method. Also, they can just override it if they want to.